light_curves/code_src/helpers/scale_up.py

```diff
@@ -107,7 +111,36 @@ def run(*, build, **kwargs_dict):
 
 
 def _build_sample(*, get_samples, consolidate_nearby_objects, sample_file, overwrite_existing_sample, **_):
-    """Build an AGN sample using coordinates from different papers."""
+    """
+    Build an AGN (Active Galactic Nuclei) sample using coordinates from different papers.
+
+    This function orchestrates the process of creating an AGN sample by fetching data from various sources,
+    cleaning the samples, and consolidating nearby objects. The resulting sample is saved to a specified file.
+
+    Parameters
+    ----------
+    get_samples : dict
+        A dictionary where keys are sample names and values are dictionaries of 
+        keyword arguments to be passed to the corresponding sample retrieval functions.
+    consolidate_nearby_objects : bool
+        If True, nearby objects will be consolidated.
+    sample_file : pathlib.Path
+        The file path where the sample will be saved.
+    overwrite_existing_sample : bool
+        If True, the existing sample file will be overwritten if it exists.
+
+    Returns
+    -------
+    astropy.table.Table
+        The resulting sample table containing the AGN objects.
+
+    Notes
+    -----
+    If the sample file already exists and `overwrite_existing_sample` is False, the function will read 
```

Comment:
We don't generate API docs from these, but overall sphinx is unhappy with unsolvable links like this, so double backticks is usually the way to go.
```suggestion
    If the sample file already exists and ``overwrite_existing_sample`` is False, the function will read 
```